src/main/scala/scynamo/ScynamoEncoder.scala

```diff
@@ -26,18 +26,14 @@ object ScynamoEncoder extends DefaultScynamoEncoderInstances {
 }
 
 trait DefaultScynamoEncoderInstances extends ScynamoIterableEncoder {
-  implicit val stringEncoder: ScynamoEncoder[String] = value =>
-    if (value.nonEmpty) Right(AttributeValue.builder().s(value).build())
-    else Either.leftNec(ScynamoEncodeError.invalidEmptyValue(ScynamoType.String))
+  implicit val stringEncoder: ScynamoEncoder[String] = value => Right(AttributeValue.builder().s(value).build())
 
   implicit val stringOptionEncoder: ScynamoEncoder[Option[String]] = {
     case Some("") | None => Right(AttributeValue.builder().nul(true).build())
     case Some(value)     => stringEncoder.encode(value)
   }
```

Comment:
In that case we can remove this
```suggestion
```

Reply:
I think this is still valuable.

Reply:
But why, the sole purpose of it was to avoid the error on empty string?

Reply:
Alright. Removed in https://github.com/moia-dev/scynamo/pull/136/commits/d1fdf899f5d96c9bab7bd6634822e4025e3b8bb8.